tests/working_directory.rs

```diff
@@ -1,4 +1,4 @@
-use super::*;
+use {super::*, fs::Permissions, std::os::unix::fs::PermissionsExt};
```

Comment:
`std::os::unix::…` is unix-only, so this import should be moved inside of the test which uses it, and the test should be set to `#[cfg(unix)] `.